{"name": "Authorized user can create a post", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n +  where 404 = <Response [404]>.status_code", "trace": "@allure.feature(\"Posts\")\n    @allure.title(\"Authorized user can create a post\")\n    def test_create_post():\n        with allure.step(\"Login and get access token\"):\n            token = test_login_success()\n    \n        headers = {\"Authorization\": f\"Bearer {token}\"}\n        post_data = {\"title\": \"Test Post\", \"content\": \"Some content\"}\n    \n        with step_request(\"POST /posts/\"):\n            response = requests.post(f\"{BASE_URL}/posts/\", headers=headers, json=post_data)\n            attach_response(response)\n    \n        with step_check_status():\n>           assert response.status_code == 200\nE           assert 404 == 200\nE            +  where 404 = <Response [404]>.status_code\n\ntests\\test_auth_flow.py:62: AssertionError"}, "steps": [{"name": "Login and get access token", "status": "passed", "steps": [{"name": "Send request: POST /auth/login", "status": "passed", "attachments": [{"name": "Status Code", "source": "85b932d3-962b-4afd-a052-c3f777f6c467-attachment.txt", "type": "text/plain"}, {"name": "Response Body", "source": "55d65cd4-077f-4d82-9528-665d194fd4a0-attachment.json", "type": "application/json"}], "start": 1753782869003, "stop": 1753782871036}, {"name": "Check response status code", "status": "passed", "start": 1753782871036, "stop": 1753782871036}, {"name": "Validate response body content", "status": "passed", "start": 1753782871036, "stop": 1753782871036}], "start": 1753782869003, "stop": 1753782871036}, {"name": "Send request: POST /posts/", "status": "passed", "attachments": [{"name": "Status Code", "source": "b6288fed-0222-4992-a510-4a4df1ce37ea-attachment.txt", "type": "text/plain"}, {"name": "Response Body", "source": "aedca0f3-cb63-4e47-898b-b09f2d94c58e-attachment.json", "type": "application/json"}], "start": 1753782871036, "stop": 1753782873096}, {"name": "Check response status code", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n +  where 404 = <Response [404]>.status_code\n", "trace": "  File \"C:\\Users\\kmish\\Desktop\\projects\\fullstack_pet_project\\tests\\test_auth_flow.py\", line 62, in test_create_post\n    assert response.status_code == 200\n"}, "start": 1753782873097, "stop": 1753782873097}], "start": 1753782869003, "stop": 1753782873099, "uuid": "4c8c7591-9ff9-4ee8-9186-f34d415d4c3c", "historyId": "a223857426b4a05bbe71b9252b056f2e", "testCaseId": "a223857426b4a05bbe71b9252b056f2e", "fullName": "tests.test_auth_flow#test_create_post", "labels": [{"name": "feature", "value": "Posts"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_flow"}, {"name": "host", "value": "LAPTOP-IKM1QBQF"}, {"name": "thread", "value": "11532-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_flow"}], "titlePath": ["tests", "test_auth_flow.py"]}